samples/exposed-spring/gradle.properties

```diff
@@ -0,0 +1 @@
+exposedVersion=0.41.1
```

Comment:
Set exposed version 0.41.1 because 0.42.0 exposed-spring-boot-starter has not found issue

Reply:
It should be now ok to bump the Exposed version to 0.43.0 (and also Kotlin versions in build.gradle.kts). Please let me know if there are still any dependency issues.